HotSpot/View Controllers/DetailsViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  DetailsViewController.m
+//  HotSpot
+//
+//  Created by drealin on 7/18/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "DetailsViewController.h"
+
+#import "Listing.h"
+#import "BookingViewController.h"
+#import "DataManager.h"
+
+@interface DetailsViewController ()
+@property (weak, nonatomic) IBOutlet UIImageView *listingImageView;
+@property (weak, nonatomic) IBOutlet UILabel *listingAddressLabel;
+@property (weak, nonatomic) IBOutlet UILabel *listingPriceLabel;
+@property (weak, nonatomic) IBOutlet UILabel *listingOwnerLabel;
+@property (weak, nonatomic) IBOutlet UILabel *listingNotesLabel;
+
+@end
+
+@implementation DetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // image
+    [DataManager getAddressNameFromPoint:self.listing.address withCompletion:^(NSString *name, NSError * _Nullable error){
+        if(error) {
+            NSLog(@"%@", error);
+        }
+        else {
+            self.listingAddressLabel.text = name;
+        }
+    }];
+    
+    
+    self.listingPriceLabel.text = [NSString stringWithFormat: @"$%@/hr", self.listing.price];
+    PFUser *homeowner = self.listing.homeowner;
+    [homeowner fetchInBackgroundWithBlock:^(PFObject * _Nullable object, NSError * _Nullable error) {
+        self.listingOwnerLabel.text = object[@"name"];
+    }];
+}
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    if([@"bookingSegue" isEqualToString:segue.identifier]) {
+        UINavigationController *navigationController = [segue destinationViewController];
```

Comment:
same comment as before. why do we have so many navigation controllers? we should have one at the beginning, and the rest should be plain view controllers. let's chat about the design on Monday?